specs/APIReview_TextureStream.md

```diff
@@ -0,0 +1,353 @@
+      [in] ICoreWebView2StagingTextureStreamTextureErrorEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+  /// Remove listener for texture error event.
+  HRESULT remove_TextureError([in] EventRegistrationToken token);
+  /// Updates d3d Device when it is updated by RenderAdapterLUIDUpdated
+  /// event.
+  HRESULT UpdateD3DDevice([in] IUnknown* d3dDevice);
+}
+/// Texture stream buffer that the host writes to so that the Renderer
+/// will render on it.
+[uuid(0836f09c-34bd-47bf-914a-99fb56ae2d07), object, pointer_default(unique)]
+interface ICoreWebView2StagingTexture : IUnknown {
+    /// Returns shared Windows NT handle. The caller expected to open it with
+    /// ID3D11Device1::OpenSharedResource1 and writes the incoming texture to it.
+    [propget] HRESULT Handle([out, retval] HANDLE* handle);
+    /// Returns IUnknown type that could be query interface to IDXGIResource.
+    /// The caller can write incoming texture to it.
+    [propget] HRESULT Resource([out, retval] IUnknown** resource);
+}
+/// This is the callback for new texture stream request.
+[uuid(62d09330-00a9-41bf-a9ae-55aaef8b3c44), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamStartRequestedEventHandler : IUnknown {
+  //// Called to provide the implementer with the event args for the
+  //// corresponding event. There are no event args and the args
+  //// parameter will be null.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] IUnknown* args);
+}
+/// This is the callback for stop request of texture stream.
+[uuid(4111102a-d19f-4438-af46-efc563b2b9cf), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamStopRequestedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event. There are no event args and the args
+  /// parameter will be null.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] IUnknown* args);
+}
+/// This is the callback for texture stream rendering error.
+[uuid(52cb8898-c711-401a-8f97-3646831ba72d), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamTextureErrorEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] ICoreWebView2StagingTextureStreamTextureErrorEventArgs* args);
+}
+/// This is the event args interface for texture stream error callback.
+[uuid(0e1730c1-03df-4ad2-b847-be4d63adf700), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamTextureErrorEventArgs : IUnknown {
+  /// Error kind.
+  [propget] HRESULT Kind([out, retval]
+      COREWEBVIEW2_TEXTURE_STREAM_ERROR_KIND* value);
+  // Texture buffer that the error is associated with.
+  HRESULT GetBuffer([out, retval] ICoreWebView2StagingTexture** buffer);
```

Comment:
This can be a property instead right? `[propget] HRESULT Buffer([out, retval] ICoreWebView2StagingTexture** value);`